sdk/token.go

```diff
@@ -646,4 +655,5 @@ type AuthSigninConsumerToken struct {
 	RedirectURI       string `json:"redirect_uri,omitempty"`
 	RequireMFA        bool   `json:"require_mfa,omitempty"`
 	IsFirstConnection bool   `json:"is_first_connection,omitempty"`
+	LinkUser          bool   `json:"link_user,omitempty"`
```

Comment:
Why no creating dedicated payload for link requests ?

Reply:
In UI, it's the same page that handle the request. I though it's easier to manage and avoid field conflicts